easybuild/easyconfigs/m/manta/manta-1.1.1.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::  Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::    Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::    MIT
+#
+# Notes::      This is a binary build and only works on 64-bit linux
+##
+
+easyblock = 'CmdCp'
+
+name = 'manta'
+version = '1.1.1'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = """A probabilistic framework for structural variant discovery.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/Illumina/manta/releases/download/v%(version)s']
+sources = ['manta-%(version)s.centos5_x86_64.tar.bz2']
+# https://github.com/Illumina/manta/releases/download/v1.1.1/manta-1.1.1.centos5_x86_64.tar.bz2
+
+skipsteps = ['build']
+
+files_to_copy = [(['bin/*'], 'bin'), 'lib', 'libexec', 'share']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['configManta.py', 'runMantaWorkflowDemo.py']],
+    'dirs': ['libexec'],
+}
+
+modextrapaths = {
+    'PATH': '%(installdir)s/libexec'
+}
```

Comment:
@robqiao You shouldn't specify the full path by using `%(installdir)s` here, just the path relative to the install directory; EasyBuild will automatically prepend the installation prefix.

@verdurin This explains the error you are seeing.

```python
modextrapaths = {
    'PATH': 'libexec',
}
```

@robqiao Maybe also include a couple of binaries located here in `sanity_check_paths`?

Reply:
Oops, my ignorance.